apps/web/src/validators/process/classification.ts

```diff
@@ -10,7 +10,7 @@ export function hasAllEmptyFields({
   publishingGroup,
 }: Pool): boolean {
   return !!(
-    empty(classifications) &&
+    !empty(classifications) &&
```

Comment:
Is this right?

Reply:
We want to show the `NullDisplay` on the "Advertisement details" section when they first appear. Instead of the Display showing all "not provided" errors. 

**Note that the empty states shown below need updating, because by default, the page shows a bunch of "Not provided" fields due to the previous page asking about classification. I'm hoping to avoid this by showing a null state instead, with their previous selections for classification and other fields applied to the "Editable" view instead.**

However, I should just remove the classifications from this function all together or create a new one without the classification check. Let me fix that. 😆

Reply:
Removed classifcation check in [5b3273f](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9029/commits/5b3273f92a42c095fb35b241824c0288e5ec623a)

Reply:
If we show a null state instead, would it be okay to check for `empty(classifications)`? Logically, it seems odd to _not_ check for it here.

Reply:
Okay, I understand now why not to check for empty classifications. Can you just rename the function and leave a comment?

Reply:
Okay changes are here: [a267460](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9029/commits/a267460cc64732ae0ae92d5c611b8a828d0376dc)